doc/config.yaml

```
@@ -48,11 +48,13 @@ params:
       column1:
         title: ""
         links:
-          - text: Scientific Python Forum
-            link: https://discuss.scientific-python.org
-      column2:
-        links:
+          - text: About
+            link: /about/teams/
```

Looking above, I think this should just be
```suggestion
            link: /about/
```